steps/install-bundler/1.0.1/step.yml

```diff
@@ -0,0 +1,31 @@
+title: Install specific bundler version
+summary: This step allows you to uninstall the system bundler version, and match the
+  one in your Gemfile.
+description: This step allows you to uninstall the system bundler version, and match
+  the one in your Gemfile..
+website: https://github.com/FutureWorkshops/bitrise-step-install-bundler
+source_code_url: https://github.com/FutureWorkshops/bitrise-step-install-bundler
+published_at: 2019-02-06T18:58:36.415567+01:00
```

Comment:
support_url cannot be removed, this is the main channel our users can contact you if they found an issue